.cicd/platforms/unpinned/macos-10.15-unpinned.sh

```diff
@@ -3,10 +3,31 @@ set -eo pipefail
 VERSION=1
 export SDKROOT="$(xcrun --sdk macosx --show-sdk-path)"
 brew update
-brew install git cmake python libtool libusb graphviz automake wget gmp pkgconfig doxygen openssl jq boost libpq libpqxx postgres || :
+brew install git cmake python libtool libusb graphviz automake wget gmp pkgconfig doxygen openssl jq   || :
 pip3 install requests
+
+# install boost from source
```

Comment:
why was boost build strategy changed here?

Reply:
Brew install boost get boost 1.76, it looks some files changed so that 1.76 can't allow eos build successfully.  I tried brew install boost@1.72, unfortunately brew can't find 172.  so, just build from source to solve that.

Reply:
The spirit of the unpinned pipeline is just that -- to catch problems a user would have when not doing a pinned build. If 1.76 is broken it needs to be fixed (not in this PR of course).